### PR TITLE
The Great Cardboard Overhaul

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
```SUMMARY: Content "Adds more cardboard boxes, cardboard forts, sheets of plastic, and more"```

#### Purpose of change
Currently the only cardboard item is `box_small`.  Cardboard boxes are ubiquitous and should be more represented in game. Also missing are sturdy sheets of flexible plastic, either 6mil for house sealing or similarly durable plastics for industrial wrapping. These go hand in hand since the two are often found in similar places.

#### Describe the solution
Adds medium and large cardboard boxes.
Large cardboard boxes can be found in many places: I have made them often spawn in place of crates.
Large boxes can also be used to make forts.
Plastic sheets spawn in many places as well, especially in hardware stores (rolls of 6mil plastic). They can be used as groundsheets much like a leather tarp.

#### Describe alternatives you've considered
I have plans to have more items spawn inside boxes, disassembling like MREs to produce a box plus contents.

#### Additional context
Another step towards making flat pack furniture, and later the beggars. Now all the pieces are in place!

### CODE REVIEW

File: data/json/construction.json [hunk 1]
```diff
@@ -2489,6 +2489,22 @@
...
+    "type": "construction",
+    "description": "Build Cardboard Fort",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 0 ] ],
+    "time": 15,
+    "pre_special": "check_empty",
+    "components": [
+      [ [ "pillow", 4 ], [ "down_pillow", 4 ] ],
+      [ [ "down_blanket", 2 ], [ "blanket", 2 ], [ "fur_blanket", 2 ] ],
+      [ [ "large_cardboard_box", 1 ] ],
```
Comment (Reclusive-reptile) [inline data/json/construction.json]:
```suggestion
      [ [ "box_large", 1 ] ],
```

File: data/json/furniture.json [hunk 2]
```diff
@@ -4279,6 +4300,119 @@
...
+    "description": "A fort built by tipping a cardboard box on its side, lining it with blankets, and partly weather sealing it with a plastic sheet.",
+    "color": "brown",
+    "move_cost_mod": 3,
+    "coverage": 95,
+    "comfort": 4,
+    "floor_bedding_warmth": 700,
+    "required_str": 3,
+    "deconstruct": {
+      "items": [
+        { "item": "large_cardboard_box", "count": 1 },
```
Comment (Reclusive-reptile) [inline data/json/furniture.json]:
```suggestion
        { "item": "box_large", "count": 1 },
```

File: data/json/furniture.json [hunk 3]
```diff
@@ -4279,6 +4300,119 @@
...
+    "symbol": "#",
+    "bgcolor": "brown",
+    "move_cost_mod": -1,
+    "coverage": 100,
+    "description": "This is a pile of cardboard boxes that have been filled with rags and junk and stacked together like bricks to form a wall.",
+    "required_str": -1,
+    "flags": [ "NOITEM", "BLOCKSDOOR", "FLAMMABLE_ASH", "ORGANIC", "EASY_DECONSTRUCT", "BLOCK_WIND" ],
+    "deconstruct": {
+      "items": [
+        { "item": "large_cardboard_box", "count": 1 },
```
Comment (Reclusive-reptile) [inline data/json/furniture.json]:
```suggestion
        { "item": "box_large", "count": 1 },
```

File: data/json/furniture.json [hunk 4]
```diff
@@ -4279,6 +4300,119 @@
...
+        { "item": "cardboard_box", "count": 2 },
```
Comment (Reclusive-reptile) [inline data/json/furniture.json]:
```suggestion
        { "item": "box_medium", "count": 2 },
```

File: data/json/furniture.json [hunk 5]
```diff
@@ -4279,6 +4300,119 @@
...
+        { "item": "small_cardboard_box", "count": 4 },
```
Comment (Reclusive-reptile) [inline data/json/furniture.json]:
```suggestion
        { "item": "box_small", "count": 4 },
```

File: data/json/furniture.json [hunk 6]
```diff
@@ -4279,6 +4300,119 @@
...
+        { "item": "plastic_sheet", "count": 2 }
+      ]
+    },
+    "bash": {
+      "str_min": 8,
+      "str_max": 30,
+      "sound": "crash!",
+      "sound_fail": "whump.",
+      "items": [
+        { "item": "large_cardboard_box", "count": [ 0, 1 ] },
```
Comment (Reclusive-reptile) [inline data/json/furniture.json]:
```suggestion
        { "item": "box_large", "count": [ 0, 1 ] },
```

File: data/json/furniture.json [hunk 7]
```diff
@@ -4279,6 +4300,119 @@
...
+        { "item": "cardboard_box", "count": [ 0, 2 ] },
```
Comment (Reclusive-reptile) [inline data/json/furniture.json]:
```suggestion
        { "item": "box_medium", "count": [ 0, 2 ] },
```

File: data/json/furniture.json [hunk 8]
```diff
@@ -4279,6 +4300,119 @@
...
+        { "item": "small_cardboard_box", "count": [ 0, 4 ] },
```
Comment (Reclusive-reptile) [inline data/json/furniture.json]:
```suggestion
        { "item": "box_small", "count": [ 0, 4 ] },
```

File: data/json/furniture.json [hunk 9]
```diff
@@ -4279,6 +4300,119 @@
...
+    "description": "A large cardboard box: this could be used to store things, or as a hiding place.",
+    "color": "brown",
+    "move_cost_mod": 7,
+    "coverage": 90,
+    "comfort": 1,
+    "floor_bedding_warmth": 200,
+    "required_str": 3,
+    "deconstruct": { "items": [ { "item": "large_cardboard_box", "count": 1 } ] },
+    "max_volume": 6000,
+    "deployed_item": "large_cardboard_box",
```
Comment (Reclusive-reptile) [inline data/json/furniture.json]:
```suggestion
    "deployed_item": "box_large",
```

**Comment by chaisawlajatang** [comment]
I feel obligated to post this.
https://i.kym-cdn.com/photos/images/original/000/544/018/e7f.jpg

**Comment by I-am-Erk** [comment]
THanks for spotting those, reclusive reptile

**Comment by Mecares** [comment]
We need a morale boost for cat mutants that are on cardboard box furniture.

**Comment by TechyBen** [comment]
> We need a morale boost for cat mutants that are on cardboard box furniture.

You mean *in*? (My cat IRL has classic DOS game boxes it sleeps in) :P

**Comment by adamkad1** [comment]
> > Large cardboard box
> 
> I feel obligated to post this.
> https://i.kym-cdn.com/photos/images/original/000/544/018/e7f.jpg

we need to be able to wear large cardboard boxes

**Pull request merged**